src/index.html

```diff
@@ -4,9 +4,312 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
```

Comment:
Logo is a part of nav, isn't it? 🤔

```suggestion
      <nav class="nav">
        <a href="#" class="logo">
          <img
            src="./images/logo.png"
            alt="Moyo logo"
            class="logo__image"
          >
        </a>
```

Reply:
> Logo is a part of nav, isn't it? 🤔

@SalehBUD Thanks for reviewing. Actually it's a standalone block that can be used anywhere in the page. Also if you want to put a nav block in the footer, you might not want a logo there.